porting-postgres.md

```diff
@@ -0,0 +1,87 @@
+---
+title: Porting from PostgreSQL
+summary: Porting an application from PostgreSQL
+toc: false
+---
+
+Although CockroachDB supports Postgres syntax and drivers, it does not offer exact compatibility. This page documents the known list of differences between Postgres and CockroachDB for identical input. That is, a SQL statement of the type listed here will behave differently than in Postgres. Porting an existing application to CockroachDB will require changing these expressions.
+
+Note that some of these differences only apply to rare inputs, and so no change will be needed, even if the listed feature is being used. In these cases, it is safe to ignore the porting instructions.
+
+<div id="toc"></div>
+
+### Overflow of `float`
+
+The `float` type in Postgres will return an error if it overflows or an expression would return Infinity:
+
+```
+postgres=# select 1e300::float * 1e10::float;
+ERROR:  value out of range: overflow
+postgres=#  select pow(0::float, -1::float);
+ERROR:  zero raised to a negative power is undefined
+```
+
+In CockroachDB these expressions will instead return Infinity:
+
+```
+root@:26257/> select 1e300::float * 1e10::float;
++----------------------------+
+| 1e300::FLOAT * 1e10::FLOAT |
++----------------------------+
+| +Inf                       |
++----------------------------+
+root@:26257/> select pow(0::float, -1::float);
++---------------------------+
+| pow(0::FLOAT, - 1::FLOAT) |
++---------------------------+
+| +Inf                      |
++---------------------------+
+```
+
+### Precedence of unary `~`
+
+The unary `~` (bitwise not) operator in Postgres has a low precedence. For example:
+
+```
```

Comment:
Rather than break the sentence with the code block, I'd rewrite as:

> For example, the following query is parsed as `~ (1 + 2)` because `~` has a lower precedence than `+`:
> ```
> SELECT ~1 + 2`